### ISSUE TITLE
SoftmaxWithLoss Operators.

### ISSUE THREAD

**Issue by lcy-seso** [PaddlePaddle/Paddle#3866]
- Combined Softmax and Cross-Entropy loss operator.
- The operator computes the softmax normalized values for each layer in the batch
of the given input, after which cross-entropy loss is computed.
- This operator is
numerically more stable than separate Softmax and CrossEntropy ops.
